package.json

```diff
@@ -100,7 +100,7 @@
     "migrate-branch": "workspace:*",
     "pnpm-utils": "workspace:*",
     "rimraf": "^3.0.2",
-    "turbo": "^1.6.1",
+    "turbo": "^1.6.3",
```

Comment:
Is it possible to revert it, I guess we don't want to put a change on this version yet or we will do it on our side